### DISCUSSION TITLE
source of error when testing async

### DISCUSSION THREAD

**Discussion by bit4bit** [babashka/nbb#331] [Q&A]
hello thanks for this project.
is there a way to get the source of error?, example

```clojure
(ns test
  (:require [cljs.test :as t :refer [async deftest is testing]]
            [promesa.core :as p]))

(defn invalid []
  (p/create (fn []
              (.invalid js/Error))))

(deftest trace-test
  (testing "source of exception?"
    (async done
           (-> (invalid)
               (p/finally done)))))

(t/run-tests)
```

output

`
Testing test
----- Error --------------------------------------
Message:  Could not find instance method: invalid

Ran 1 tests containing 0 assertions.
0 failures, 0 errors.
`

thanks any help

**Comment by borkdude** [comment]
Thanks for posting this. I'll see what I can do. This has to do with how SCI (the interpreter works).
